dhctl/pkg/preflight/dhctl_obsolescence.go

```diff
@@ -123,7 +123,7 @@ func (pc *PreflightCheck) fetchDeckhouseImageHashFromReleaseChannel(ctx context.
 		return nil, fmt.Errorf("parse ClusterConfiguration.deckhouse.registryDockerCfg: %w", err)
 	}
 
-	imageReference, err := name.ParseReference(pc.installConfig.GetImage())
```

Comment:
why are not we using for preflight check force using tag?

Reply:
Why compare with the image of the release channel? When we insert an end tag into the deployment. If the final goal is that no updates occur during installation, then we need to compare both the image tag for the tag and for the release channel, if there is one. In general, the release channel planned to transfer the config from the init configuration to the config module and remove it from the init configuration.

Reply:
>to transfer the config from the init configuration to the config module and remove it from the init configuration.
and then it becomes optional altogether